jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/SelectsTests.java

```diff
@@ -74,7 +74,7 @@ public void readOnlySelectTest() {
         readonlySelect.is().collapsed();
     }
 
-    @Test
+    @Test(description = "Test checks that only first selected value is shown and others are counted")
     public void selectionSelectTest() {
         selectionSelect.show();
         selectionSelect.select("buzz");
```

Comment:
do we have disabled tests in Select? :(

Reply:
You mean appendAndPrependItemSelectTest() ?

Reply:
yes, appendAndPrependItemSelectTest()